lambda/README.md

```diff
@@ -0,0 +1,57 @@
+# Running a lambda on localstack
+
```

Comment:
This required a `go get` for `github.com/aws/aws-lambda-go/events` and `github.com/aws/aws-lambda-go/lambda` for me. Can that be added in step 1 here?